src/platform_impl/windows/event_loop.rs

```diff
@@ -214,11 +254,11 @@ impl<T: 'static> EventLoop<T> {
 
             runner.poll();
             'main: loop {
-                if 0 == winuser::GetMessageW(&mut msg, ptr::null_mut(), 0, 0) {
+                if !GetMessageW(&mut msg, 0, 0, 0) == true.into() {
```

Comment:
Semantic change (positive comparison to `true` instead of negative comparison to `false`)

Also, this one is a bit confusing to read